docs/UserGuide.md

```diff
@@ -456,7 +456,7 @@ Action | Format, Examples
 -------|------------------------------
 **Update**| `s-update A/NUM B/NUM C/NUM ...` <br> e.g., `s-update BSBM/100 BSBBT/120`
 **List**| `s-list`
-**Find**| `s-find KEYWORD [MORE_KEYWORDS] ...` <br> e.g., `s-find BSBM BSBBT`
+**Find**| `s-find KEYWORD [MORE_KEYWORDS]` <br> e.g., `s-find BSBM BSBBT`
```

Comment:
"..." should not be removed, as user can key in multiple keywords for this command.